package training.learn.exceptons

/** This exception is used to indicate that lesson is interrupted by another lesson or by some other reason */
class LessonScriptStopped : Exception() {
}